src/components/Tooltip/index.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+const Tooltip = styled.div`
+    display: ${props => (props.active ? 'block' : 'none')};
+    position: absolute;
+    top: -52px;
```

Comment:
These values (`top` and `left`) must be dynamic. because if I use a huge text (Or an `img`) inside the Tooltip it will be misaligned

I suggest use `bottom: 120%` instead this `top` property.